plugins/woocommerce/includes/class-wc-customer-download.php

```diff
@@ -311,14 +311,15 @@ public function track_download( $user_id = null, $user_ip_address = null ) {
 WHERE permission_id = %d",
 			$this->get_id()
 		);
-		$wpdb->query( $query ); // WPCS: unprepared SQL ok.
+		// phpcs:ignore WordPress.DB.PreparedSQL.NotPrepared
+		$wpdb->query( $query );
 
 		// Re-read this download from the data store to pull updated counts.
 		$this->data_store->read( $this );
 
 		// Track download in download log.
 		$download_log = new WC_Customer_Download_Log();
-		$download_log->set_timestamp( current_time( 'timestamp', true ) );
+		$download_log->set_timestamp( time() );
```

Comment:
unexpected change?

Reply:
This change has been done by phpcs. The outputs seem to be identical in both cases.

Reply:
Isn't `current_time()` actually a [WordPress function](https://developer.wordpress.org/reference/functions/current_time/)? And it takes into account the timezone of the site, so it will not be the same necessarily as `time()`

Reply:
Ah nvm, since the second param is true, it will return in GMT, which should end up being the same as `time()`